용재/1406.py

```diff
@@ -0,0 +1,27 @@
+# https://www.acmicpc.net/problem/1406
+# 에디터
+
+getText = list(input())
+
+N = int(input())
+
+inputNumbers = [str(input()) for _ in range(N)]
+
+L = len(getText)
+
+restText = []
+
+for i in inputNumbers:
+  if i[0] == "L":
+    if len(getText) > 0:
+      restText.append(getText.pop())
+  elif i[0] == "D":
+    if len(restText) > 0:
+      getText.append(restText.pop())
+  elif i[0] == "B":
+    if len(getText) > 0:
+      getText.pop()
+  elif i[0] == "P":
+    getText.append(i[2])
```

Comment:
저도 이렇게 비교적 간단한 문제에서는 인덱스를 바로 쓰고는 합니다만

아래와 같이 해당 인덱스가 뜻하는 바를 이름으로 하여 변수를 선언해 둔다면 훨씬 가독성이 좋아지더라구요!

(물론 저도 쓰는 버릇을 들여야 합니다 ㅎㅎ..)

```suggestion
COMMAND, CHAR = 0, 2
for i in inputNumbers:
  if i[COMMAND] == "L":
    if len(getText) > 0:
      restText.append(getText.pop())
  elif i[COMMAND] == "D":
    if len(restText) > 0:
      getText.append(restText.pop())
  elif i[COMMAND] == "B":
    if len(getText) > 0:
      getText.pop()
  elif i[COMMAND] == "P":
    getText.append(i[CHAR])
```